File: Empty/java/src/main/java/com/gmail/l2t45s7e9/java/interactor/ContactDetailsInteractor.java
package com.gmail.l2t45s7e9.java.interactor;

import com.gmail.l2t45s7e9.java.entity.Contact;
import io.reactivex.rxjava3.core.Single;
public interface ContactDetailsInteractor {
    Single<Contact> getContactDetailsRepo(String id, int color);
}

File: Empty/java/src/main/java/com/gmail/l2t45s7e9/java/interactor/ContactDetailsInteractor.kt
package com.gmail.l2t45s7e9.java.interactor

import com.gmail.l2t45s7e9.java.entity.Contact
import kotlinx.coroutines.flow.Flow

interface ContactDetailsInteractor {
    suspend fun getContactDetailsRepo(id: String?, color: Int): Flow<Contact?>
}